### ISSUE TITLE
PrimeNG: autocomplete:  Mobile VoiceOver on iOS A11y Issue

### ISSUE THREAD

**Issue by jpfaehler** [primefaces/primefaces#15060]
### Describe the bug

It appears that the iOS Mobile VoiceOver is unable to navigate autocomplete component dropdown options through the typical gestures most users would use for accessibility requirements.

### Reproducer

This is reproducible on the PrimeNG 22 demo site (https://primeng.dev/autocomplete) by using an iOS Mobile Voiceover iPhone. 

Steps to repro:

1. Turn on Mobile VoiceOver (easiest is to say "Hey Siri, turn on VoiceOver"
2. Focus on an autocomplete element by tapping it once
3. Voiceover will readout instruction to doubletap the autocomplete to expand the options list
4. Double tap the autocomplete and notice no options appear and voiceover reads that your in a dead end.

The same behavior happens if you try to double tap the arrow button on the auto complete element.

### Expected behavior

Expected behavior is that the options list would open and the user can continue to work through the UI using VoiceOver.

### PrimeFaces edition

None

### PrimeFaces version

PrimeNG 22

### JSF implementation

Mojarra

### JSF version

26.5.2

### Java version

n/a

**Comment by melloware** [comment]
@jpfaehler  this is for PrimeFaces JSF not for PrimeNG. Please use the forums...
